examples/src/main/java/com/mapbox/navigation/examples/activity/OnboardRouterActivityJava.java

```diff
@@ -0,0 +1,229 @@
+package com.mapbox.navigation.examples.activity;
+
+import android.os.Bundle;
+import android.os.Environment;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.material.snackbar.Snackbar;
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
+import com.mapbox.geojson.Point;
+import com.mapbox.mapboxsdk.annotations.MarkerOptions;
+import com.mapbox.mapboxsdk.camera.CameraUpdateFactory;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapView;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+import com.mapbox.mapboxsdk.maps.OnMapReadyCallback;
+import com.mapbox.mapboxsdk.maps.Style;
+import com.mapbox.navigation.base.route.Router;
+import com.mapbox.navigation.examples.R;
+import com.mapbox.navigation.route.onboard.MapboxOnboardRouter;
+import com.mapbox.navigation.route.onboard.model.Config;
+import com.mapbox.services.android.navigation.ui.v5.route.NavigationMapRoute;
+
+import java.io.File;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+//import com.mapbox.navigation.base.route.DirectionsSession;
+//import com.mapbox.navigation.directions.session.MapboxDirectionsSession;
+
+public class OnboardRouterActivityJava
```

Comment:
Examples need to be cleaned up, they're also not added to the examples list.

Reply:
where can I find _examples list_?

Reply:
https://github.com/mapbox/mapbox-navigation-android/blob/master/examples/src/main/java/com/mapbox/navigation/examples/MainActivity.java#L55-L71 :)

Reply:
I see, thanks)